// bloomer-extensions/dist/App.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

require('bulma');

require('@fortawesome/fontawesome-free/css/fontawesome.css');

var _bloomer = require('bloomer');

var _components = require('./components');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }

function _possibleConstructorReturn(self, call) { if (!self) { throw new ReferenceError("this hasn't been initialised - super() hasn't been called"); } return call && (typeof call === "object" || typeof call === "function") ? call : self; }

function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }

var App = function (_Component) {
  _inherits(App, _Component);

  function App() {
    var _ref;

    var _temp, _this, _ret;

    _classCallCheck(this, App);

    for (var _len = arguments.length, args = Array(_len), _key = 0; _key < _len; _key++) {
      args[_key] = arguments[_key];
    }

    return _ret = (_temp = (_this = _possibleConstructorReturn(this, (_ref = App.__proto__ || Object.getPrototypeOf(App)).call.apply(_ref, [this].concat(args))), _this), _this.state = {
      isBurgerActive: false,
      quickview: true
    }, _temp), _possibleConstructorReturn(_this, _ret);
  }

  _createClass(App, [{
    key: 'render',
    value: function render() {
      var _this2 = this;

      return _react2.default.createElement(
        _react.Fragment,
        null,
        _react2.default.createElement(
          _bloomer.Navbar,
          { style: { border: 'solid 1px #00D1B2', margin: '0' } },
          _react2.default.createElement(
            _bloomer.NavbarBrand,
            null,
            _react2.default.createElement(
              _bloomer.NavbarItem,
              null,
              'Bloomer-extensions'
            ),
            _react2.default.createElement(
              _bloomer.NavbarItem,
              { isHidden: 'desktop' },
              _react2.default.createElement(_bloomer.Icon, { className: 'fa fa-github' })
            ),
            _react2.default.createElement(_bloomer.NavbarBurger, { isActive: this.state.isBurgerActive, onClick: function onClick() {
                return _this2.setState({ isBurgerActive: !_this2.state.isBurgerActive });
              } })
          ),
          _react2.default.createElement(
            _bloomer.NavbarMenu,
            { isActive: this.state.isActive, onClick: this.onClickNav },
            _react2.default.createElement(
              _bloomer.NavbarStart,
              null,
              _react2.default.createElement(
                _bloomer.NavbarItem,
                { href: '#/' },
                'Home'
              ),
              _react2.default.createElement(
                _bloomer.NavbarItem,
                { hasDropdown: true, isHoverable: true },
                _react2.default.createElement(
                  _bloomer.NavbarLink,
                  { href: '#/documentation' },
                  'Other Projects'
                ),
                _react2.default.createElement(
                  _bloomer.NavbarDropdown,
                  null,
                  _react2.default.createElement(
                    _bloomer.NavbarItem,
                    { href: 'https://bulma.io' },
                    'Bulma'
                  ),
                  _react2.default.createElement(
                    _bloomer.NavbarItem,
                    { href: 'https://wikiki.github.io' },
                    'Bulma extensions'
                  ),
                  _react2.default.createElement(_bloomer.NavbarDivider, null),
                  _react2.default.createElement(
                    _bloomer.NavbarItem,
                    { href: 'https://jesobreira.github.io/Bulma-Form-Builder/' },
                    'Bulma Form Builder'
                  )
                )
              )
            ),
            _react2.default.createElement(
              _bloomer.NavbarEnd,
              null,
              _react2.default.createElement(
                _bloomer.NavbarItem,
                { href: 'https://github.com/AlgusDark/bloomer', isHidden: 'touch' },
                _react2.default.createElement(_bloomer.Icon, { className: 'fa fa-github' })
              ),
              _react2.default.createElement(
                _bloomer.NavbarItem,
                { href: 'https://twitter.com/AlgusDark', isHidden: 'touch' },
                _react2.default.createElement(_bloomer.Icon, { className: 'fa fa-twitter', style: { color: '#55acee' } })
              ),
              _react2.default.createElement(
                _bloomer.NavbarItem,
                null,
                _react2.default.createElement(
                  _bloomer.Field,
                  { isGrouped: true },
                  _react2.default.createElement(
                    _bloomer.Control,
                    null,
                    _react2.default.createElement(
                      _bloomer.Button,
                      { id: 'twitter', 'data-social-network': 'Twitter', 'data-social-action': 'tweet',
                        'data-social-target': 'http://bloomer.js.org', target: '_blank', href: 'https://twitter.com/intent/tweet?text=bloomer: a set of React Stateless Components for bulma.io&url=http://bloomer.js.org&via=AlgusDark' },
                      _react2.default.createElement(_bloomer.Icon, { className: 'fa fa-twitter' }),
                      _react2.default.createElement(
                        'span',
                        null,
                        'Tweet'
                      )
                    )
                  )
                )
              )
            )
          )
        ),
        _react2.default.createElement(
          _bloomer.Hero,
          { isColor: 'primary', isSize: 'small' },
          _react2.default.createElement(
            _bloomer.HeroBody,
            null,
            _react2.default.createElement(
              _bloomer.Container,
              null,
              _react2.default.createElement(
                _bloomer.Title,
                null,
                'Bloomer Extensions'
              ),
              _react2.default.createElement(
                _bloomer.Subtitle,
                null,
                'React components for bulma-extensions'
              )
            )
          ),
          _react2.default.createElement(
            _bloomer.HeroFooter,
            null,
            _react2.default.createElement(
              _bloomer.Tabs,
              null,
              _react2.default.createElement(
                _bloomer.Container,
                null,
                _react2.default.createElement(
                  _bloomer.TabList,
                  null,
                  _react2.default.createElement(
                    _bloomer.Tab,
                    { isActive: true },
                    _react2.default.createElement(
                      _bloomer.TabLink,
                      null,
                      'Overview'
                    )
                  ),
                  _react2.default.createElement(
                    _bloomer.Tab,
                    null,
                    _react2.default.createElement(
                      _bloomer.TabLink,
                      null,
                      'Layout'
                    )
                  ),
                  _react2.default.createElement(
                    _bloomer.Tab,
                    null,
                    _react2.default.createElement(
                      _bloomer.TabLink,
                      null,
                      'Form'
                    )
                  ),
                  _react2.default.createElement(
                    _bloomer.Tab,
                    null,
                    _react2.default.createElement(
                      _bloomer.TabLink,
                      null,
                      'Elements'
                    )
                  ),
                  _react2.default.createElement(
                    _bloomer.Tab,
                    null,
                    _react2.default.createElement(
                      _bloomer.TabLink,
                      null,
                      'Components'
                    )
                  )
                )
              )
            )
          )
        ),
        _react2.default.createElement(
          _bloomer.Container,
          null,
          _react2.default.createElement(_components.Divider, null),
          _react2.default.createElement(_components.Divider, { isVertical: true }),
          _react2.default.createElement(_components.Divider, { content: 'HI' }),
          _react2.default.createElement(
            _components.PageLoader,
            { isActive: true },
            'Test'
          ),
          _react2.default.createElement(
            _components.ToolTip,
            { text: 'Hey' },
            'Test'
          ),
          _react2.default.createElement(
            _components.Badge,
            { text: '8' },
            _react2.default.createElement(
              'button',
              { 'class': 'button' },
              'Test'
            )
          ),
          _react2.default.createElement(
            _components.CheckRadio,
            { id: 'test', checked: this.state.checkradioChecked, onClick: function onClick() {
                return _this2.setState({ checkradioChecked: !_this2.state.checkradioChecked });
              } },
            'Click me'
          ),
          _react2.default.createElement(
            _components.CheckRadio,
            { type: 'radio', name: 'myGroup', id: 'opt1', value: '1', checked: true },
            'Option 1'
          ),
          _react2.default.createElement(
            _components.CheckRadio,
            { type: 'radio', name: 'myGroup', id: 'opt2', value: '2' },
            'Option 2'
          ),
          _react2.default.createElement(_components.Slider, { value: this.state.sliderValue, onChange: function onChange(e) {
              return _this2.setState({ sliderValue: e.target.value });
            } }),
          this.state.sliderValue || 50,
          _react2.default.createElement(
            _components.CheckSwitch,
            { checked: this.state.checkswitchChecked, onClick: function onClick() {
                return _this2.setState({ checkswitchChecked: !_this2.state.checkswitchChecked });
              } },
            'Check me'
          ),
          _react2.default.createElement(
            _components.QuickView,
            { isActive: this.state.quickview },
            _react2.default.createElement(
              _components.QuickViewHeader,
              null,
              'Title',
              _react2.default.createElement(_bloomer.Delete, { onClick: function onClick() {
                  return _this2.setState({ quickview: false });
                } })
            ),
            _react2.default.createElement(
              _components.QuickViewBody,
              null,
              _react2.default.createElement(
                _components.QuickViewBlock,
                null,
                'Hello, world'
              )
            ),
            _react2.default.createElement(
              _components.QuickViewFooter,
              null,
              'Footer'
            )
          ),
          _react2.default.createElement(
            _components.Steps,
            { isSize: 'large' },
            _react2.default.createElement(
              _components.StepItem,
              { isCompleted: true, isColor: 'info' },
              _react2.default.createElement(
                _components.StepMarker,
                null,
                '\u2713'
              ),
              _react2.default.createElement(
                _components.StepDetails,
                null,
                _react2.default.createElement(
                  _components.StepTitle,
                  null,
                  'Step 1'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Steps\' first step'
                )
              )
            ),
            _react2.default.createElement(
              _components.StepItem,
              { isActive: true },
              _react2.default.createElement(
                _components.StepMarker,
                null,
                '\u2713'
              ),
              _react2.default.createElement(
                _components.StepDetails,
                null,
                _react2.default.createElement(
                  _components.StepTitle,
                  null,
                  'Step 2'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Steps\' second step'
                )
              )
            ),
            _react2.default.createElement(
              _components.StepItem,
              null,
              _react2.default.createElement(
                _components.StepMarker,
                null,
                '\u2713'
              ),
              _react2.default.createElement(
                _components.StepDetails,
                null,
                _react2.default.createElement(
                  _components.StepTitle,
                  null,
                  'Step 3'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Steps\' third step'
                )
              )
            ),
            _react2.default.createElement(
              _components.StepItem,
              null,
              _react2.default.createElement(
                _components.StepMarker,
                null,
                '\u2713'
              ),
              _react2.default.createElement(
                _components.StepDetails,
                null,
                _react2.default.createElement(
                  _components.StepTitle,
                  null,
                  'Step 4'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Steps\' final step'
                )
              )
            )
          ),
          _react2.default.createElement(
            _components.Timeline,
            null,
            _react2.default.createElement(
              _components.TimelineHeader,
              null,
              _react2.default.createElement(
                _bloomer.Tag,
                { isSize: 'medium', isColor: 'primary' },
                'Start'
              )
            ),
            _react2.default.createElement(
              _components.TimelineItem,
              null,
              _react2.default.createElement(_components.TimelineMarker, null),
              _react2.default.createElement(
                _components.TimelineContent,
                null,
                _react2.default.createElement(
                  'p',
                  { 'class': 'heading' },
                  'January 2016'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Timeline content'
                )
              )
            ),
            _react2.default.createElement(
              _components.TimelineItem,
              null,
              _react2.default.createElement(_components.TimelineMarker, { isColor: 'info', isSize: 'medium' }),
              _react2.default.createElement(
                _components.TimelineContent,
                null,
                _react2.default.createElement(
                  'p',
                  { 'class': 'heading' },
                  'January 2017'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Timeline content'
                )
              )
            ),
            _react2.default.createElement(
              _components.TimelineItem,
              null,
              _react2.default.createElement(
                _components.TimelineMarker,
                { isColor: 'info', isSize: 'medium', isIcon: true },
                '\u2713'
              ),
              _react2.default.createElement(
                _components.TimelineContent,
                null,
                _react2.default.createElement(
                  'p',
                  { 'class': 'heading' },
                  'January 2017'
                ),
                _react2.default.createElement(
                  'p',
                  null,
                  'Timeline content'
                )
              )
            )
          )
        )
      );
    }
  }]);

  return App;
}(_react.Component);

exports.default = App;
