// bloomer-extensions/dist/components/CheckSwitch/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
	value: true
});
exports.CheckSwitch = CheckSwitch;

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

require('bulma-switch');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }

function CheckSwitch(_ref) {
	var props = _objectWithoutProperties(_ref, []);

	return _react2.default.createElement(
		'div',
		{ 'class': 'field' },
		_react2.default.createElement('input', {
			id: props.id,
			type: 'checkbox',
			name: props.name,
			checked: props.checked,
			value: props.value,
			disabled: props.disabled,
			className: 'switch\n\t\t  \t\t\t' + (props.isColor && "is-" + props.isColor) + '\n\t\t\t\t\t' + (props.isSize && "is-" + props.isSize) + '\n\t\t\t\t\t' + (props.isRtl && "is-rtl") + '\n\t\t\t\t\t' + (props.isThin && "is-thin") + '\n\t\t\t\t\t' + (props.isRounded && "is-rounded") + '\n\t\t\t\t\t' + (props.isOutlined && "is-outlined") + '\n\t\t  \t\t\t'
		}),
		_react2.default.createElement(
			'label',
			{
				onClick: props.onClick,
				htmlFor: props.id },
			props.children
		)
	);
}
