// bloomer-extensions/dist/components/Slider/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
	value: true
});
exports.Slider = Slider;

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

require('bulma-slider/dist/css/bulma-slider.min.css');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }

function Slider(_ref) {
	var props = _objectWithoutProperties(_ref, []);

	return _react2.default.createElement('input', { className: 'slider\n\t\t\t' + (props.isColor && "is-" + props.isColor) + '\n\t\t\t' + (props.isSize && "is-" + props.isSize) + '\n\t\t\t' + (props.isCircle && "is-circle") + '\n\t\t\t' + (props.isFullwidth && "is-fullwidth") + '\n\t\t\t',
		step: props.step || 1,
		min: props.min || 0,
		max: props.max || 100,
		value: props.value || 50,
		type: 'range',
		disabled: props.disabled,
		onChange: props.onChange,
		orient: props.orient || "horizontal"
	});
}
