// bloomer-extensions/dist/components/CheckRadio/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
	value: true
});
exports.CheckRadio = CheckRadio;

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

require('bulma-checkradio');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }

function CheckRadio(_ref) {
	var props = _objectWithoutProperties(_ref, []);

	return _react2.default.createElement(
		'div',
		{ 'class': 'field' },
		_react2.default.createElement('input', {
			id: props.id,
			type: props.type || "checkbox",
			name: props.name,
			checked: props.checked,
			value: props.value,
			onChange: props.onChange,
			disabled: props.disabled,
			className: 'is-checkradio\n\t\t  \t\t\t' + (props.isColor && "is-" + props.isColor) + '\n\t\t\t\t\t' + (props.isSize && "is-" + props.isSize) + '\n\t\t\t\t\t' + (props.isRtl && "is-rtl") + '\n\t\t\t\t\t' + (props.isCircle && "is-circle") + '\n\t\t\t\t\t' + (props.isBlock && "is-block") + '\n\t\t\t\t\t' + (props.hasNoBorder && "has-no-border") + '\n\t\t\t\t\t' + (props.hasBackgroundColor && "has-background-color") + '\n\t\t  \t\t\t'
		}),
		_react2.default.createElement(
			'label',
			{ htmlFor: props.id },
			props.children
		)
	);
}
